src/components/list/EntryList.js

```diff
@@ -35,6 +45,8 @@ const EntryList = forwardRef(
     { locations, onEntryClick, onEntryMouseEnter, onEntryMouseLeave, ...props },
     ref,
   ) => {
+    const settings = useSelector((state) => state.settings)
+    console.log(settings)
```

Comment:
Here you want to select `distanceUnit` directly. That way StyledListEntry won't just re-render on any settings change, only for distanceUnit.